File: Heat/Models/CausalWarehouse.vb
''' <summary>
''' Rappresenta una causale di movimento di magazzino.
''' </summary>
''' <remarks></remarks>
Public Class CausalWarehouse
    Property ID As Integer
    Property Code As String
    Property Sign As Integer

    Property TypeID As Integer
    Property Type As CausalWarehouseTypeEnum
    Property Transaction As Boolean

End Class

File: Heat.ConvertedToC#/Models/CausalWarehouse.cs
namespace Heat.Models
{
    /// <summary>
    /// Rappresenta una causale di movimento di magazzino.
    /// </summary>
    /// <remarks></remarks>
    public class CausalWarehouse
	{
		public int ID { get; set; }
		public string Code { get; set; }
		public int Sign { get; set; }

		public int TypeID { get; set; }
		public CausalWarehouseTypeEnum Type { get; set; }
		public bool Transaction { get; set; }

	}
}
